boilerplate/flyte/golang_support_tools/tools.go

```diff
@@ -1,3 +1,4 @@
+//go:build tools
```

Comment:
Should this be dropped? It looks like it was auto-generated.

Reply:
This is ok...